### PR TITLE
Include doc link in K8s component registration event

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@zakisk do you think that you can help with the MeshKit Error Codes Utility failure?

File: server/handlers/k8sconfig_handler.go [hunk 1]
```diff
@@ -334,7 +334,9 @@ func RegisterK8sMeshModelComponents(provider *models.Provider, _ context.Context
 		}, c)
 		count++
 	}
-	event := events.NewEvent().ActedUpon(connectionUUID).WithCategory("kubernetes_components").WithAction("registration").FromSystem(mesheryInstanceID).FromUser(userUUID).WithSeverity(events.Informational).WithDescription(fmt.Sprintf("%d Kubernetes components registered for %s", count, ctxName)).Build()
+	event := events.NewEvent().ActedUpon(connectionUUID).WithCategory("kubernetes_components").WithAction("registration").FromSystem(mesheryInstanceID).FromUser(userUUID).WithSeverity(events.Informational).WithDescription(fmt.Sprintf("%d Kubernetes components registered for horse %s", count, ctxName)).WithMetadata(map[string]interface{}{
```
Comment (MUzairS15) [inline server/handlers/k8sconfig_handler.go:337]:
```suggestion
	event := events.NewEvent().ActedUpon(connectionUUID).WithCategory("kubernetes_components").WithAction("registration").FromSystem(mesheryInstanceID).FromUser(userUUID).WithSeverity(events.Informational).WithDescription(fmt.Sprintf("%d Kubernetes components registered for %s", count, ctxName)).WithMetadata(map[string]interface{}{
```
Host/Context name?

File: server/handlers/k8sconfig_handler.go [hunk 2]
```diff
@@ -334,7 +334,9 @@ func RegisterK8sMeshModelComponents(provider *models.Provider, _ context.Context
...
+		"doc": "https://docs.meshery.io/tasks/lifecycle-management",
```
Comment (MUzairS15) [inline server/handlers/k8sconfig_handler.go:338]: @aabidsofi19 is Metadata formatter in UI dynamic to handle this?

Comment (aabidsofi19) [reply server/handlers/k8sconfig_handler.go:338]: @MUzairS15  yeah it is  .

Comment (MUzairS15) [reply server/handlers/k8sconfig_handler.go:338]: Yes, I just saw your PR.

File: server/handlers/k8sconfig_handler.go (hunk 1 above)
Comment (aabidsofi19) [reply server/handlers/k8sconfig_handler.go:337]: @leecalcote  @MUzairS15  for most of metadata properties , i have noticed we are using capital words , so should we use `Doc` here .

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]

**Pull request merged**